### PR TITLE
Proposed governance changes from the branching discussions

### PR DESCRIPTION
**Pull request by jskeet**
The intention is that this covers the *content* changes required as part of #954.

(There may be other files I'm not aware of... happy to be educated and take another pass.)

### CODE REVIEW

File: community/GOVERNANCE.md [hunk 1]
```diff
@@ -141,36 +141,39 @@ The specifications produced will adhere to the following:
...
   - Add some descriptive text, or the list of PRs that have been merged since
     the previous release. The git query to get the list commits since the last
     release is:
     `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`.
     Just replace "v0.1" with the name of the previous release.
   - Press `Publish release` button
-- Create a PR that modifies the version string in all of the files
-  (but not the README.md table) to be the next version number with a `-wip`
-  in it (e.g `v0.4-wip`).
-- Merge the PR.
```
Comment (duglin) [inline community/GOVERNANCE.md:173]: I think we may need a PR to modify the table in the main README to identify the latest release of each "subject", don't we? The table shows the "latest/wip" version and the "latest/released" versions for each spec.

Comment (jskeet) [reply community/GOVERNANCE.md:173]: Right. It's not quite the same as it was before, but I agree we need something... adding.

Comment (jskeet) [reply community/GOVERNANCE.md:173]: Ah no, it's already in the "Create a PR that modifies the readme". Basically there's *always* latest/wip (just "the main branch") and latest/released (modified by the release PR). We don't need a separate PR afterwards to "reset" state to WIP anyway, as far as I can see.

**Comment by duglin** [comment]
> I wasn't sure whether we'd want to actually close it, as there's the web site aspect too. Happy to go with the flow though.

Good point - I'll remove it

File: community/GOVERNANCE.md (hunk 1 above)
Comment (duglin) [reply community/GOVERNANCE.md:173]: ah I missed that. cool!

File: community/GOVERNANCE.md [hunk 2]
```diff
@@ -141,36 +141,39 @@ The specifications produced will adhere to the following:
...
   \*.md files) that include a version string, to the new release version string.
   Make sure to remove `-wip` from all of the version strings.
 - Merge the PR.
 - Create a [new release](https://github.com/cloudevents/spec/releases/new):
-  - Choose a "Tag version" of the form: `vX.Y`, e.g. `v0.1`
+  - Choose a "Tag version" of the form: `<subject>-vX.Y.Z`, e.g.
+    `cloudevents-v1.0.4` or `discovery-v1.0.0`
   - Target should be `main`, the default value
-  - Release title should be the same as the Tag - `vX.Y`
+  - Release title should be the same as the Tag - `<subject>-vX.Y.Z`
```
[the remark this answers is not in the document]
Comment (jskeet) [reply community/GOVERNANCE.md:170]: Sounds fine - I'll make this change and emphasize that the subject can also be `x/y/z`. Will try to get to that on Monday.

[the remark this answers is not in the document]
Comment (jskeet) [reply community/GOVERNANCE.md]: Done now - PTAL.

File: community/GOVERNANCE.md [hunk 3]
```diff
@@ -141,36 +141,40 @@ The specifications produced will adhere to the following:
...
-  will eventually result in an official "patch" release (e.g. `1.0.2`)
 
 Note that these rules do not apply to the
 [documented extensions](../cloudevents/documented-extensions.md).
 
+All versions are tagged from the `main` branch, but the tag only applies to
+the "subject" of the release - the directory containing the information
+covered by that release (e.g. `discovery` or `cloudevents`). The
+[CloudEvents web site](https://cloudevents.io/) takes appropriate content from
+each tagged version. (If the directory containing the information covered
```
Comment (duglin) [inline community/GOVERNANCE.md:152]: I'm having trouble parsing the sentence in parens. Did you leave off a "is in a nested sub-directory"  after "by the release" ?

Comment (jskeet) [reply community/GOVERNANCE.md:152]: Whoops, yes. Fixing.

**Comment by duglin** [comment]
Approved on the 3/18 call

**Pull request merged**